### PR TITLE
[PE-1874] Adds Python Sequr demo

### PR DESCRIPTION
**Pull request by quantropi-minh**

### CODE REVIEW

**Review by quantropi-dlou — APPROVED** [review]

File: Python/readme.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# QiSpace Python Demos
+
+## Sequr
+
+### Setup
+  1. Place your Qeep `.whl` library file under `lib` folder. If you do not have one, please contact Quantropi Inc.
```
Comment (quantropi-Alexis) [inline Python/readme.md]: 'Qeep' all in caps?

File: Python/readme.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  1. Install demo dependencies and libraries: `./install.sh`
+  1. Have your qispace meta config `.json` file from Quantropi Inc ready. If you do not have one, please contact Quantropi Inc.
+
+### Running the demo
+
+  1. **Generate key demo:**
+      - Execute the `demo_sequr_key_gen.py` with the bellow aruments:
+        ```
```
Comment (quantropi-Alexis) [inline Python/readme.md:14]: 'bellow aruments' misspelled

File: Python/readme.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# QiSpace Python Demos
+
+## Sequr
+
+### Setup
+  1. Place your Qeep `.whl` library file under `lib` folder. If you do not have one, please contact Quantropi Inc.
+  1. Install demo dependencies and libraries: `./install.sh`
+  1. Have your qispace meta config `.json` file from Quantropi Inc ready. If you do not have one, please contact Quantropi Inc.
```
Comment (quantropi-Alexis) [inline Python/readme.md]: 'qispace' in all caps?

File: Python/readme.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      - You might get an output as bellow:
+        ```
+        ------------------------
+        Key generation successful
+        Key ID: 60d27860-3be1-4eae-92a3-2c10129eeb60
+        Key: 86 83 b1 57 a2  ...  b7 90 83 65 fe
+        ------------------------
+        ```
+  1. **Query key demo:**
+      - Execute the `demo_sequr_key_query.py` with the bellow aruments:
```
Comment (quantropi-Alexis) [inline Python/readme.md]: 'bellow aruments' misspelled

File: Python/readme.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        usage: ./demo_sequr_key_gen.py [-h] --qispace_meta QISPACE_META [--key_size_bits KEY_SIZE_BITS]
+
+        options:
+          -h, --help            show this help message and exit
+          --qispace_meta QISPACE_META
+                                Path to qispace meta .json file, provided by Quantropi Inc
+          --key_size_bits KEY_SIZE_BITS
+                                Key size to generate (in bits, defaults: 256)
+        ```
+      - You might get an output as bellow:
```
Comment (quantropi-Alexis) [inline Python/readme.md]: 'bellow' misspelled

**Review by quantropi-Alexis — CHANGES_REQUESTED** [review]

**Pull request merged**